core/imageroot/var/lib/nethserver/node/etc/support/nft.conf

```diff
@@ -0,0 +1,19 @@
+#
+# support-nft.conf -- NFT NAT configuration for the support VPN
+#
+# Connections to some ports are forwarded to the local host. This NFT
+# configuration is applied in a separate network namespace started with
+# slirp4netns, where IP 10.0.2.2 is then mapped to 127.0.0.1
+#
+table ip nat {
+        chain postrouting {
+                type nat hook postrouting priority srcnat; policy accept;
+                masquerade
+        }
+
+        chain prerouting {
+                type nat hook prerouting priority dstnat; policy accept;
+                iifname tun0 tcp dport { 22, 80, 443, 9090 } dnat to 10.0.2.2
```

Comment:
Change the tun name also here.
Also probably this rule will not work if SSH is not listing to port 22.